src/doc/gui_manual/Contributing/index.rst

```diff
@@ -20,14 +20,14 @@ You can then browse the root of the manual by pointing your browser to
 :file:`./_build/index.html`.  The ``-a`` forces a re-build of everything.
 Remove it when you are constantly revising and rebuilding.
 
-Your changes to any files in :file:`trunk/docs/SphinxDocs` will go live
+Your changes to any files ``.rst`` files in :file:`visit/src/doc` will go live
```

Comment:
Change "any files" to "any".

Reply:
Fixed. Sorry for all the individual commits. Was doing  via browser directly on GitHub. That doesn't work too well for these large files.